wrappers/vroom.rb

```diff
@@ -282,7 +282,7 @@ def collect_jobs(vrp, vrp_skills, vrp_units)
             ((q&.value || 0) * CUSTOM_QUANTITY_BIGNUM).round
           }
         }.delete_if{ |_k, v|
-          v.nil? || v.is_a?(Array) && v.empty?
+          !v || v.is_a?(Array) && v.empty?
```

Comment:
Does this related to the fix? It looks like a bug to me -- with `!v` if we ever add a field with a value `false` to this hash, it will be removed --  I find v.nil? more expressive and to the point.